src/libs/DateUtils.js

```diff
@@ -324,6 +422,83 @@ function getStatusUntilDate(inputDate) {
     return translateLocal('statusPage.untilTime', {time: format(input, `${CONST.DATE.FNS_FORMAT_STRING} ${CONST.DATE.LOCAL_TIME_FORMAT}`)});
 }
 
+/**
+ * Update the hour and minute of a date.
+ *
+ * @param {string} inputTime - Time in "hh:mm A" format (like "10:55 AM").
```

Comment:
I wonder if this should be `updatedTime` to make things a bit clearer here (which one is updating which).

Reply:
Yes, it would be better, thanks!